src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card"
+      data-qa="card"
+    >
+
+      <div class="card__image">
+        <img
+        class="card__image-iMac"
+        src="images/imac.jpeg"
+        alt="iMac"
+        >
+      </div>
+
+      <div class="card__title">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
+
+      <div class="card__code">
+        <p>Product code: 195434</p>
+      </div>
+
+      <div class="card__rate">
+        <div class="card__star stars-4">
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
```

Comment:
The number of active stars should be set with a modifier. Please see the project page hints section p2 and Checklist page p3.
My suggestion: 
instead of block star-4 you can convert it to modifier stars--4 and use 'star' block along with it. as The block modifier can only be added to the block itself. An element can only have a modifier for that element
can go like this-
<div class="card__rate">
        <div class="stars stars--4">